packages/manager/apps/web/client/app/hosting/multisite/add/hosting-multisite-add.controller.js

```diff
@@ -42,23 +42,29 @@ angular
       $scope.MAX_DOMAIN_LENGTH = WucValidator.MAX_DOMAIN_LENGTH;
 
       $scope.isStep1Valid = () => {
-        if (!$scope.model.options && !$scope.model.needChangeOffer) {
+        if (
+          !$scope.model.options &&
+          !$scope.model.moreThanOneDomainCountWithOneAttached
```

Comment:
**nitpick:** `moreThanOneDomainCountWithOneAttached` is pretty had to understand. Can you shorten the name?